### ISSUE TITLE
Add explicit way to create new file (a.k.a. wizard for new class/worksheet)

### ISSUE THREAD

**Issue by alekseiAlefirov** [scalameta/metals-vscode#179]
The option to create new scala file or worksheet (or, potentially, notebook) should be added to
- most expected, explorer directory context menu,
and, additionally can be added to
- file menu (not sure if there's such an option in VS Code)
- command palette

**Should there be actual wizard?**
Where wizard is a separate window with controls. It suppose something like IDEa's wizard to choose among Scala class/object/trait.
- Should there be such choice at all?
- Additionally, such choice assume auto creating blank class/object/trait with user-suggested name, which can conflict with package provider, triggered by `didOpen`, editor-independent.
- Not sure, there's a way to create wizards in VS Code (quick googling does not give positive answer on that).

As a first iteration my suggestion is to create a simple explorer context menu addition under `New File`:
```
|- New File
|- New Scala >
|- |- File (or Class, not sure which is more accurate here)
|- |- Worksheet
|-...
```

**Comment by tgodzik** [comment]
@gabro  Sorry for not warning you about it!

**Comment by olafurpg** [comment]
I think this is a good idea. I would avoid making this functionality limited to VS Code. It’s vest to implement most of the logic in the Metals server. We already have metals/inputBox that I believe is also supported in Vim.

**Comment by olafurpg** [comment]
One VS Code api we don’t have an extension for right now is “quick open” (I think it’s called that). I’m not sure we need it for this feature however. We could add a server command “New Scala worksheet”. We might be able to expose this command in the context menu in VS Code but it should also be possible to invoke the command from other editors.

**Comment by ckipp01** [comment]
Closed via #183
